examples/using-remark-copy-linked-files/package.json

```diff
@@ -18,7 +18,7 @@
     "gatsby-source-filesystem": "^2.0.1",
     "gatsby-transformer-remark": "^2.1.1",
     "gatsby-transformer-sharp": "^2.1.1",
-    "hast-util-to-html": "^3.1.0",
+    "hast-util-to-html": "^4.0.0",
```

Comment:
This probably isn't needed at all if it's _already_ a dependency of gatsby-transformer-remark and we depend on that?

Reply:
Should be safe to remove, yes.